test/print/print.ml

```diff
@@ -16,7 +16,7 @@ struct
       List ((Atom "compilation_unit")::(Atom name)::hidden)
 
   let root : Root.t -> sexp = fun {package; file; digest} ->
-    List [Atom package; odoc_file file; Atom digest]
+    List [Atom package; odoc_file file; Atom (Digest.to_hex digest)]
```

Comment:
Sorry, I'm not sure why this is needed. It also seems like the tests are failing because of this with `[invalid] Digest.to_hex`.

Reply:
@rizo I fixed that in `master`. Only the second commit in this PR is properly part of the PR, the first one was rewritten in `master`.